platform/functionality/field_capabilities/field_caps.go

```diff
@@ -70,8 +70,9 @@ func handleFieldCapsIndex(cfg map[string]config.IndexConfiguration, schemaRegist
 			for fieldName, field := range fieldsWithAliases {
 				addFieldCapabilityFromSchemaRegistry(fields, fieldName.AsString(), field.Type, resolvedIndex)
 				switch field.Type.Name {
-				case "keyword", "text":
+				case "text":
```

Comment:
Seems like "text", should be swapped with "keyword", at least looking at capabilities we are adding